tests/scripts/thread-cert/Cert_5_8_02_KeyIncrement.py

```diff
@@ -85,56 +85,55 @@ def verify(self, pv):
 
         # Step 1: The DUT must start the network using
         # thrKeySequenceCounter = 0
-        _lpkts = leader_pkts.filter_mle_cmd(consts.MLE_ADVERTISEMENT).must_next()
+        _lpkts = leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
```

Comment:
`must_next()` returns a single Packet
```suggestion
        lpkt = leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
```

BTW, what does `lpkts` stands for?